src/ImageRunModal.jsx

```diff
@@ -737,6 +752,7 @@ export class ImageRunModal extends React.Component {
                                   </button>
                               </Popover>
                           }
+                          helperText={Object.keys(this.props.registries).length === 0 ? _("No search registries configured in /etc/containers/registries.conf") : ""}
```

Comment:
@martinpitt @garrett I am not 100% satisfied with this text, but it is the simplest I could thing of.

Reply:
I'd honestly just leave this out. The UI provides a fallback, the text is not actionable ("so what do I do? is this an error?" → it's not), and IMHO we don't really want to encourage people to change the default config file.

IMHO it is significantly more important to make it easy and discoverable to search in a new registry which is specified right in the dialog.

If you really want to keep this, please change the fallback to `: undefined`, to avoid rendering an empty string as helper text (possibly the component already DTRT, but it still looks cleaner that way). The standard pattern is `length > 0 && _("..")`

Reply:
Yeah, it isn't an error and the registries we have by default are fine. We could, as an enhancement, later add a popover somewhere that contains text something to the effect of:

> Searching for images uses system-configured registries in `/etc/containers/registries.conf`. If no registries are configured, `docker.io` and `quay.io` will be used as defaults.

Or we could just leave it out, which is what we probably should do for now.

We can and probably should mention it in the release notes, however.

Reply:
Yeah, on trying this locally, it really feels like there's an error with that text (even though it's not indicated as an error or warning). It's better to just leave it out, at least for now.

Reply:
Ok! I have left it out, it also feels weird to say, "No configured registries, falling back to docker.io, quay.io".